comps/embeddings/multimodal_embeddings/multimodal_langchain/requirements.txt

```diff
@@ -0,0 +1,15 @@
+docarray[full]
+fastapi
+huggingface_hub
+langchain
+langsmith
```

Comment:
please remove langsmith

Reply:
Thank for your feedback. We have removed langsmith here.